docs/volto_customization/installation.md

```diff
@@ -17,150 +17,124 @@ Getting started with Volto involves setting up a development environment, unders
-yo @plone/volto:addon
+make install
 ```
 
-Note: You can also use the namespace like `@plone-collective/volto-teaser-tutorial` (or any other) is not required and is
-optional. We're using namespaces for scoped package under some organisation.
+## Start Plone
 
-Use `volto-teaser-tutorial` as the package name. After the
-scaffolding of the add-on completes, you can check the created files in
-`src/addons/volto-teaser-tutorial`.
+Plone 6 has two servers: one for the frontend, and one for the backend.
+As such, we need to maintain two active shell sessions, one for each server, to start your Plone site.
 
-Back to the project, you can edit `jsconfig.json` and add your add-on:
+### Start Plone backend
 
-```json
-{
-  "compilerOptions": {
-    "baseUrl": "src",
-    "paths": {
-      "volto-teaser-tutorial": ["addons/volto-teaser-tutorial/src"]
-    }
-  }
-}
-```
+In the currently open session, issue the following command.
 
-```{note}
-The `jsconfig.json` file is needed by Volto to identify development
-packages. You are not strictly limited to Volto add-ons in its use, you
-could, for example, use this to make it easier to debug third-party
-JavaScript packages that are shipped transpiled.
+```shell
+make backend-start
 ```
 
-### Volto addon script
+### Start Plone frontend
 
-Alternatively, if you already have an addon pushed to a remote repository and you want to create a volto development stack with it, you can use our addon script to easily scaffold a dev environment without creating a project externally.
+Create a second shell session in a new window.
+Start the Plone frontend with the following command.
 
 ```shell
-npx -p @plone/scripts addon clone [options] <source> [destination]
+make frontend-start
 ```
 
-This command downloads the volto-teaser-tutorial add-on from its git repository's main branch, and will generate a project with the latest Volto version.
+Open a browser at the following URL to visit your Plone site.
 
-```shell
-npx -p @plone/scripts addon clone https://github.com/kitconcept/volto-teaser-tutorial.git --branch main
-```
+http://localhost:3000
 
-### (Optional) Use mrs-developer to sync add-on to GitHub
+You can then login with admin/admin at <a target="_blank" href="http://localhost:3000/login">http://localhost:3000/login</a>.
 
-You can also immediately push the package to GitHub, then use `[mrs-developer]`
-to manage the package and `jsconfig.json` changes.
+## Volto addon
 
-Install mrs-developer as a development dependency by running:
+Using Cookieplone we should already have a working Volto project with provided addon. You can find the addon in packages/volto-teaser-tutorial.
 
-```shell
-yarn add -W -D mrs-developer
+```{note}
+You might have noticed that we have `volto.config.js` in the root of the project. This is the volto configuration file allowing us to configure Volto and register addons.The addons list points to the addon we just installed. Cookieplone takes care of registering the addon for us.
```

Comment:
`{file}` renders with semantic HTML to designate it as a file, as opposed to an inline literal. I'd suggest doing that wherever you have files.

Also use care to capitalize Volto as needed.

```suggestion
You might have noticed that we have {file}`volto.config.js` in the root of the project. This is the Volto configuration file allowing us to configure Volto and register add-ons. The add-ons list points to the add-on we just installed. Cookieplone takes care of registering the add-on for us.
```